src/dsverifier.cpp

```diff
@@ -2429,17 +2477,20 @@ void verify_state_space_overshoot(void)
       x0(i, j) = _controller.x0[i][j];
     }
   }
-
+  std::cout << "Test"<< std::endl;
```

Comment:
Can you please remove these debug messages?

Reply:
Done!